modules/dnds/1.0/src/R/run_dnds.R

```diff
@@ -0,0 +1,70 @@
+### Run R package dNdS to determine the significantly mutated genes ###
+# Currently supports only hg19-based maf files
+# but future versions can be adopted to work with other genome builds.
+
+#!/usr/bin/env Rscript
+#
+# Notes:
+#   This script is intended for use with the snakefile for dNdS in LCR-modules.
+#   It uses the snakemake objects to find maf file.
+#   Additionaly, it can support the genes of targeted panel to restrict SMG search.
+#
+
+# Load packages -----------------------------------------------------------
+message("Loading packages...")
+suppressWarnings(
+suppressPackageStartupMessages({
+    library(dndscv)
+})
+)
+
+# Read the maf file
+maf = read.table(
```

Comment:
would fread be better in this case?

Reply:
I opted for the base R alternative here to reduce the number of dependencies and make the conda environment more light. But in Rstudio will definitely opt for fread or read_tsv 😄